tests/unit_tests/test_cli.py

```diff
@@ -0,0 +1,153 @@
+BASIC_MERGED_MESH = {"config":{"mesh_info":{"region":{"lat_min":-10,"lat_max":0,"long_min":-10,"long_max":10,"start_time":"2000-01-01","end_time":"2000-12-31","cell_width":10,"cell_height":10},"data_sources":[],"splitting":{"split_depth":1,"minimum_datapoints":5},"merged":[{"region":{"lat_min":0,"lat_max":10,"long_min":-10,"long_max":10,"start_time":"2000-01-01","end_time":"2000-12-31","cell_width":10,"cell_height":10},"data_sources":[],"splitting":{"split_depth":1,"minimum_datapoints":5}}]}},"cellboxes":[{"geometry":"POLYGON ((-10 -10, -10 0, 0 0, 0 -10, -10 -10))","cx":-5,"cy":-5,"dcx":5,"dcy":5,"id":"0"},{"geometry":"POLYGON ((0 -10, 0 0, 10 0, 10 -10, 0 -10))","cx":5,"cy":-5,"dcx":5,"dcy":5,"id":"1"},{"geometry":"POLYGON ((-10 0, -10 10, 0 10, 0 0, -10 0))","cx":-5,"cy":5,"dcx":5,"dcy":5,"id":"2"},{"geometry":"POLYGON ((0 0, 0 10, 10 10, 10 0, 0 0))","cx":5,"cy":5,"dcx":5,"dcy":5,"id":"3"}],"neighbour_graph":{"0":{"1":[3],"2":[1],"3":[],"4":[],"-1":[],"-2":[],"-3":[],"-4":[2]},"1":{"1":[],"2":[],"3":[],"4":[],"-1":[],"-2":[0],"-3":[2],"-4":[3]},"2":{"1":[],"2":[3],"3":[1],"4":[0],"-1":[],"-2":[],"-3":[],"-4":[]},"3":{"1":[],"2":[],"3":[],"4":[1],"-1":[0],"-2":[2],"-3":[],"-4":[]}}}
+
+
+def json_dict_to_file(json_dict, filename):
+    """
+    Converts a dictionary to a JSON formatted file
+
+    Args:
+        json_dict (dict): Dict to write to JSON
+        filename (str): Path to file being written
+    """
+    with open(filename, 'w') as fp:
+        json.dump(json_dict, fp, indent=4)
+
+def file_to_json_dict(filename):
+    """
+    Reads in a JSON file and returns dict of contents
+
+    Args:
+        filename (str): Path to file to be read
+
+    Returns:
+        dict: Dictionary with JSON contents
+    """
+    with open(filename, 'r') as fp:
+        json_dict = json.load(fp)
+    return json_dict
+
+class TestCLI (unittest.TestCase):
+
+    def setUp(self):
+        # Create temporary files to write into 
+        self.output_base_directory = tempfile.mkdtemp()
+        self.tmp_config_file = tempfile.NamedTemporaryFile()
+        self.tmp_mesh_file   = tempfile.NamedTemporaryFile()
+        self.tmp_mesh_file_1 = tempfile.NamedTemporaryFile()
+        self.tmp_mesh_file_2 = tempfile.NamedTemporaryFile()
+        self.tmp_merge_file  = tempfile.NamedTemporaryFile()
+        self.tmp_output_file = tempfile.NamedTemporaryFile()
+
+    def tearDown(self):
+        # Remove temporary files upon test completion
+        self.tmp_config_file.close()
+        self.tmp_mesh_file.close()
+        self.tmp_mesh_file_1.close()
+        self.tmp_mesh_file_2.close()
+        self.tmp_merge_file.close()
+        self.tmp_output_file.close()
+    
+    def test_get_args_cli(self):
```

Comment:
Should this (and the other similar tests) be removed if you are not planning to actually create a test for it? Or simply have it pass with the addition of a TODO, if you think we should add tests later? Currently the warnings make the output a bit messy when running through these new tests...

Reply:
For now, I'm in favour of removing the warning statement and replacing it with a TODO comment. I don't think adding the warning message saying it's not implemented adds much functionality, and the TODO makes it clear that this will be implemented in the future.